simple/src/MediumProblem/TaxCalculator.java

```diff
@@ -0,0 +1,52 @@
+package MediumProblem;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class TaxCalculator {
+    private float tax;
+
+    public float IncomeTax(float income) {
+        int slab = (income < 500000) ? 0 : (500000 <= income && income <= 700000)
+                ? 10 : (700000 <= income && income <= 1000000) ? 15 : (1000000 < income && income >= 1500000) ? 30 : 20;
```

Comment:
Conditions are not as per the problem statement

Reply:
change the logic